### DISCUSSION TITLE
socketio.exceptions.ConnectionError: Connection error

### DISCUSSION THREAD

**Discussion by yathartharora** [miguelgrinberg/python-socketio#736] [Q&A]
Hii,

I have checked the compatibility versions and everything mentioned in the previous issues, but I am still facing this error..

Here I have the correct version: 
![Screenshot 2021-05-06 at 3 05 48 PM](https://user-images.githubusercontent.com/43466752/117276517-9aba2400-ae7c-11eb-89f0-d6ae94980b9e.png)

**Comment by miguelgrinberg** [comment]
I need to see your logs please. See the troubleshooting section of the docs if you don't know how to enable logs.
